<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>800</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1280</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Archery Workout Manager v0.1</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="LBL_title">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>10</y>
      <width>771</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cantarell</family>
      <pointsize>36</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Archery Workout Manager v0.1</string>
    </property>
   </widget>
   <widget class="QLabel" name="LBL_logo">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>210</y>
      <width>461</width>
      <height>391</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>GFX/bow-and-arrow.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>100</y>
      <width>703</width>
      <height>622</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="LYT_main">
     <item row="3" column="0">
      <widget class="QPushButton" name="BTN_exManager">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Gestione Esercizi</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QPushButton" name="BTN_athlManager">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Gestione Atleti</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QLabel" name="LBL_exManager">
       <property name="font">
        <font>
         <family>Technic</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true"></string>
       </property>
       <property name="text">
        <string>Esercizi Presenti:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QLabel" name="LBL_athlManager">
       <property name="font">
        <font>
         <family>Technic</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Atleti Presenti:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QPushButton" name="BTN_wkoutManager">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Gestione Allenamenti</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="LBL_wkoutManager">
       <property name="font">
        <font>
         <family>Technic</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Piani Allenamento Presenti: </string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QLabel" name="LBL_numWkouts">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Numero Piani&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QLabel" name="LBL_numAthletes">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Numero Atleti&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item row="3" column="3">
      <widget class="QLabel" name="LBL_numEx">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Numero Esercizi&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>27</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionConnect_to_Remote_DB"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
    <addaction name="actionHelp"/>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuAbout"/>
  </widget>
  <action name="actionHelp">
   <property name="text">
    <string>Help</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionConnect_to_Remote_DB">
   <property name="text">
    <string>Connect to Remote DB</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
